.eslintrc

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Yes, it's on the list to be changed when we switch everything over to just use Prettier in defaults 2.0